gen/main/java/testmodel/micronaut_pojo/InheritanceWithEnumType1.java

```diff
@@ -32,7 +32,7 @@ public boolean equals(Object object) {
 
 	@Override
 	public int hashCode() {
-		return java.util.Objects.hash(d, super.hashCode());
```

Comment:
Which version is preferred by you @sschnabe ? super.hashCode() as part of java.util.Objects.hash() or separate?

Reply:
I would prefer it as a port of `Objects.hash()`